src/Domain/DomainCreators/AlignedBlocks.hpp

```diff
@@ -0,0 +1,93 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <array>
+#include <cstddef>
+#include <limits>
+#include <vector>
+
+#include "DataStructures/Index.hpp"
+#include "Domain/Domain.hpp"
+#include "Domain/DomainCreators/DomainCreator.hpp"
+#include "Options/Options.hpp"
+#include "Utilities/MakeArray.hpp"
+#include "Utilities/TMPL.hpp"
+
+namespace DomainCreators {
+
+/// \ingroup DomainCreatorsGroup
+/// \brief Create a Domain consisting of multiple aligned Blocks.
+///
+/// This is useful for setting up problems with piecewise smooth initial data.
+///
+/// \note Adaptive mesh refinement can never join blocks.
```

Comment:
I would give a little more context as to what prompted this note - on its own it leaves me wondering why I'm seeing this note on only this DomainCreator and not others. Is this note contrasting how this Domain would behave with the how the alternative Domain of a single Block with multiple elements would behave, under AMR?